hw/ip/kmac/doc/_index.md

```diff
@@ -199,7 +199,33 @@ In addition to that, the KMAC/SHA3 blocks the software access to the Keccak stat
 
 ### KeyMgr Interface
 
-_TBD_
+![](keymgr-intf.svg)
```

Comment:
does the main block diagram need to be updated also? It looks like in that one the key manager data is still being muxed post message fifo, whereas in the keymgr interface diagram it shows keymgr going into the message fifo.

Reply:
Ah.. good point. Let me revise the main block diagram also.

Reply:
revised. Let me merge after passing Checks.